style.css

```diff
@@ -2646,6 +2646,44 @@ div.comment:first-of-type {
 /*	13. Site Pagination
 /* --------------------------------------------------------------------------------------------- */
 
+.pagination .nav-links {
+	display: flex;
+	flex-wrap: wrap;
+	font-size: 1.8rem;
+	padding-top: 3rem;
+	font-weight: 600;
+}
```

Comment:
Moving the top padding of `.nav-links` to the wrapping `.navigation.pagination` element, and adding `margin: -2.5rem 0 0 -2.5rem` and the equivalent positive values to the children of the `.nav-links` will allow us to have vertical margins between lines without any excess margins, and horizontal margins between items without an initial left margin on the first element of each wrapping line (as in the image below).

<img width="438" alt="image" src="https://user-images.githubusercontent.com/8181091/64547539-0b6a0f80-d32d-11e9-88f5-ffe94bc6152e.png">

With this, you can use the follow targeting and CSS to get the margin between the child elements, reducing the overall amount of CSS a bit:

`.nav-links > * {
    margin: 2.5rem 0 0 2.5rem;
}`

This would also require the equivalent changes in the media query CSS.

Edit: Oh, and this will reduce the horizontal margin between the items a bit as well and remove the righthand margin from the last item in a row, slightly reducing the risk for line breaks altogether :)